packages/attention/src/component.tsx

```diff
@@ -164,17 +172,7 @@ export function Attention(props: AttentionProps) {
   useAutoUpdatePosition(targetElRef, isShowing, attentionEl, attentionState);
 
   return !props.callout && initialTargetEl === undefined ? null : (
-    <div
-      data-testid="attention-el"
-      className={classNames(
-        {
-          [ccAttention.notCallout]: !props.callout,
-          invisible: !isVisible && !props.callout,
-          hidden: !isVisible && !props.tooltip,
-        },
-        className,
-      )}
-      ref={attentionEl}>
+    <div data-testid="attention-el" className={containerClasses} ref={attentionEl}>
       <div
         role={props.role === '' ? undefined : props.tooltip ? 'tooltip' : 'img'}
         aria-label={ariaLabel === '' ? undefined : ariaLabel ?? defaultAriaLabel()}
```

Comment:
It's not introduced in this PR but the translations are not working after the i18n calls were moved to `/packages/_helpers/attention.tsx`.

Reply:
Updated in these commits: [8e6c70c](https://github.com/warp-ds/react/pull/274/commits/8e6c70c66b1ac7e5e306cd55a0da250a9b765f85) and [3b29cbb](https://github.com/warp-ds/react/pull/274/commits/3b29cbb736f59b69888ec013f845e5ccbfc3e67d) 🙌